[2025-04-20T00:27:20.727+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pdf_processing_embedding_dag.generate_embeddings_task manual__2025-04-20T00:27:08.179680+00:00 [queued]>
[2025-04-20T00:27:20.738+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pdf_processing_embedding_dag.generate_embeddings_task manual__2025-04-20T00:27:08.179680+00:00 [queued]>
[2025-04-20T00:27:20.738+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-04-20T00:27:20.750+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): generate_embeddings_task> on 2025-04-20 00:27:08.179680+00:00
[2025-04-20T00:27:20.754+0000] {standard_task_runner.py:57} INFO - Started process 99 to run task
[2025-04-20T00:27:20.759+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'pdf_processing_embedding_dag', 'generate_embeddings_task', 'manual__2025-04-20T00:27:08.179680+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/Text_processor.py', '--cfg-path', '/tmp/tmpzxrvohfc']
[2025-04-20T00:27:20.762+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask generate_embeddings_task
[2025-04-20T00:27:20.810+0000] {task_command.py:410} INFO - Running <TaskInstance: pdf_processing_embedding_dag.generate_embeddings_task manual__2025-04-20T00:27:08.179680+00:00 [running]> on host 26eb3b761e27
[2025-04-20T00:27:20.872+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pdf_processing_embedding_dag' AIRFLOW_CTX_TASK_ID='generate_embeddings_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T00:27:08.179680+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-20T00:27:08.179680+00:00'
[2025-04-20T00:27:20.888+0000] {logging_mixin.py:149} INFO - Error al generar embedding para documento 1: {"error":"model \"nomic-embed-text\" not found, try pulling it first"}
[2025-04-20T00:27:20.890+0000] {logging_mixin.py:149} INFO - Error al generar embedding para documento 2: {"error":"model \"nomic-embed-text\" not found, try pulling it first"}
[2025-04-20T00:27:20.893+0000] {logging_mixin.py:149} INFO - Error al generar embedding para documento 3: {"error":"model \"nomic-embed-text\" not found, try pulling it first"}
[2025-04-20T00:27:20.893+0000] {logging_mixin.py:149} INFO - Generados embeddings para 0 documentos
[2025-04-20T00:27:20.894+0000] {python.py:183} INFO - Done. Returned value was: {'docs_with_embeddings': [], 'doc_ids': [1, 2, 3]}
[2025-04-20T00:27:20.915+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=pdf_processing_embedding_dag, task_id=generate_embeddings_task, execution_date=20250420T002708, start_date=20250420T002720, end_date=20250420T002720
[2025-04-20T00:27:20.929+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-04-20T00:27:20.946+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
